src/lwc/rd2ElevateCreditCardForm/__tests__/rd2ElevateCreditCardForm.test.js

```diff
@@ -0,0 +1,145 @@
+import { createElement } from 'lwc';
+import rd2ElevateCreditCardForm from 'c/rd2ElevateCreditCardForm';
+import { registerSa11yMatcher } from "@sa11y/jest";
+import getOrgDomainInfo from '@salesforce/apex/UTIL_AuraEnabledCommon.getOrgDomainInfo';
+
+const PATH_GE_TOKENIZE_CARD = '/apex/GE_TokenizeCard';
+const DISABLED_MESSAGE = 'c.RD2_ElevateDisabledMessage';
+const ALERT_MESSAGE = 'Test alert message';
+
+const createWidget = () => {
+    let element = createElement(
+        'c-rd2-elevate-credit-card-form',
+        { is: rd2ElevateCreditCardForm }
+    );
+    return element;
+}
+
+jest.mock(
+    '@salesforce/apex/UTIL_AuraEnabledCommon.getOrgDomainInfo',
+    () => {
+        return {
+            default: jest.fn()
+        };
+    },
+    { virtual: true }
+);
+
+const GET_ORG_DOMAIN_ERROR = {
+    body: { message: ALERT_MESSAGE },
+    ok: false,
+    status: 400,
+    statusText: 'Bad Request'
+};
+
+function flushPromises() {
```

Comment:
`flushPromises` is defined in [jest.setup.js](https://github.com/SalesforceFoundation/NPSP/blob/master/jest.setup.js). Things put on the global jest object don't need to get defined in the context of an individual testfile.